### DISCUSSION TITLE
Random animal name list should be configurable

### DISCUSSION THREAD

**Discussion by mathieumd** [sheltermanager/asm3#1355] [Ideas]
Default random name list is nice for English-speaking countries, but I'm afraid here in France, some of [these names](https://github.com/sheltermanager/asm3/blob/master/src/asm3/animalname.py) would be hard or even impossible to pronounce for my beloved colleagues! ;-)

It would be nice if it could be editable by site admin.

As a side note, it seems there is a custom to name animals with a name whose first letter is defined by year ([by example in France](https://jardinage.lemonde.fr/dossier-1407-nom-chien.html)). Maybe it would be fun to support this? Although I have no more information about it, and it may well be quite different letters by country!

**Comment by bobintetley** [comment]
That's an interesting idea. You're right that realistically we should have implemented the random name list as a file that can be overridden to the file of your choice in asm3.conf. I'll add that to our dev list. 

The year thing is an interesting idea. I guess it means when an animal returns you know exactly what year it was in your care (assuming the new owners didn't change the name).

**Comment by bobintetley** [comment]
this was just committed as #1356 (pretty trivial change!)
